app/views/papers/edit.html.erb

```diff
@@ -3,4 +3,3 @@
 <%= render 'form', paper: @paper, button_name: "Update Paper" %>
 
 <%= link_to 'Show', @paper %> |
```

Comment:
I can still access this page, should we get rid of it temporarily in the controller?  I don't mind having it so I can just edit things if need be without going through console.